CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Lotus changelog
 
-# v 1.25.0-rc3 / 2023-11-02
+# v 1.25.0-rc4 / 2023-11-08
 
-This is the second release candidate of the upcoming OPTIONAL release Lotus v1.25.0. This optional release also supports the Filecoin network version 21 upgrade, codenamed Watermelon 🍉, in addition to the numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers.
+This is the fifth release candidate of the upcoming OPTIONAL release Lotus v1.25.0. This optional release also supports the Filecoin network version 21 upgrade, codenamed Watermelon 🍉, in addition to the numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers.
```

Comment:
4th

Reply:
Resolved in: https://github.com/filecoin-project/lotus/pull/11400/commits/677dfde7de10b729d7c4dec02b914416d8f6fd52